src/components/Deck/Deck.jsx

```diff
@@ -1,11 +1,15 @@
-// import Card from "@/components/Card/Card";
-// import "./Deck.scss";
-// import cardsData from "@/data/fakeCards.json";
+import Card from "@/components/Card/Card";
+import "./Deck.scss";
+import cardsData from "@/data/fakeCards.json";
 
-// const Deck = () => {
-// you will need to map over the cardsData array and render a Card component for each card object
-// how will you pass props to the Card component?
+const Deck = () => {
+    return (
+      <div className="deck">
+        {cardsData.map((cardData, i) => (
+          <Card title={cardData.title} url={cardData.url} image={cardData.image} key={i} />
```

Comment:
Good work with the props :) 

This doesn't really matter in this situation (it would only really matter when you're dealing with large amounts of components), but the key shouldn't really be an index. Have a read through the section here under "Keeping List Items in Order with Key" https://react.dev/learn/rendering-lists

Reply:
yes, id would be better solution 👍 Thank you!

Reply:
Well done  @RbAvci